week-1-ts-ecosystem/w01e01-typescript-basics/src/model.ts

```diff
@@ -0,0 +1,11 @@
+import type {IUser} from "./types/user";
+
+export const user: IUser = {
+	name: 'Andy',
+	age: 30,
+	email: 'andy@mail-me-tommorow.com',
+	address: {
+		street: 'Strange Alley',
+		no: 23,
+	},
+}
```

Comment:
Modelem jest `IUser` to co tutaj mamy to dane, uzupełniające model

Reply:
Przeniosłem do katalogu data